Source/Scene/Scene.js

```diff
@@ -3052,8 +3114,14 @@ define([
         tryAndCatchError(this, time, update);
         this._postUpdate.raiseEvent(this, time);
 
+        this._frustumChanged = !this._camera.frustum.equals(this._cameraClone.frustum);
+        if (this._frustumChanged && this._logDepthBuffer && !(this._camera.frustum instanceof OrthographicFrustum || this._camera.frustum instanceof OrthographicOffCenterFrustum)) {
+            this._camera.frustum.near = 1.0;
```

Comment:
Not sure about all the context here, but why is this set to `1.0`?  It is configuration and I really want to decrease the default for classes like the historic site point cloud.